PDOAssignment/LoginPage.php

```diff
@@ -0,0 +1,30 @@
+<?php
+	include 'DatabaseConfig.php';
+	if(!$databaseConnection)
+	{
+		echo "Connection Failed";
+	}
+	else
+	{
+		$flag="false";
+		$userName=$_POST['emailId'];
+		$password=$_POST['password'];
+		$selectStatement= "SELECT * FROM users WHERE email = '".$userName."' and password='".$password."'"; 
+		$statement = $databaseConnection->query($selectStatement);
+		foreach($databaseConnection->query($selectStatement) as $row)
```

Comment:
when assigned this result in the variable **$statement** then why didn't you use that variable in the loop.